.gitignore

```diff
@@ -20,3 +20,7 @@ packages/integration-tests/.env
 # custom node_modules for testing
 !packages/@lwc/module-resolver/src/__tests__/**/node_modules/
 !packages/@lwc/rollup-plugin/src/__tests__/**/node_modules/
+
+# dev mode caches
+tsconfig.tsbuildinfo
+.rollup.cache
```

Comment:
We can change where TypeScript/`@rollup/plugin-typescritpt` write their cache files, but I'm not sure it's worth it since it would be extra config. These files are not in `dist` and so they wouldn't get published to npm.

Reply:
I checked locally and `tsconfig.tsbuildinfo` is generated in `dist` for all the packages invoking directly tsc. For example: `packages/@lwc/template-compiler/dist/commonjs/tsconfig.tsbuildinfo`